src/taipy/gui/gui.py

```diff
@@ -1680,11 +1680,14 @@ def _register_data_accessor(self, data_accessor_class: t.Type[_DataAccessor]) ->
 
     def get_flask_app(self) -> Flask:
         """Get the internal Flask application.
+        - Cannot be called before the `(Gui.)run^` method has been called.
```

Comment:
You MUST add an empty line in order to isolate the short doc.

=> """Get the internal Flask application.

This method must be called \*\*after\*\* `(Gui.)run^` method was invoked.
..."""